bucket/ds4windows.json

```
@@ -3,6 +3,10 @@
     "description": "Allows to get the best experience while using a DualShock 4 on PC.",
     "homepage": "https://ryochan7.github.io/ds4windows-site/",
     "license": "MIT",
+    "notes": "ViGEmBus is required for DS4Windows to work.",
```

```suggestion
    "notes": "ViGEmBus is required for DS4Windows to work. You can install it within the app.",
```
I think this extra info would help.